### DISCUSSION TITLE
Move multiple mods to different categories

### DISCUSSION THREAD

**Discussion by mk56-spn** [ppy/osu#19219] [Ideas]
With the fun category seemingly becoming a bit of a dumping ground for new mods (which i guess makes sense but idk doesnt seem good long term ) heres a few that would probably do well to be moved 

personally i feel Traceable is no different in nature than hidden ( that is, a difficulty increasing visual only mod , that is also not a gimmick) as such i feel it is misplaced in the fun category.

No Scope also belongs in difficulty increase for obvious reasons ( im aware that it can be cheesed with extrnal os cursors, but at the very least it doesnt change difficulty, and for most people it increases it massively )

Barrel roll also never makes the game easier

Transform makes the game quantifiably  considerably harder

Repel makes the game harder but im  unsure as to whether it should be moved due to its difficulty being tied to player movements, although i guess this also applies to flashlight to a degree

Move approach different to conversion

**Comment by Moltanicaa** [comment]
I have a similar suggestion - to give the ability to change the order and location of mods on the screen for choosing them.

**Reply by peppy** [reply]
The closest we'd every consider implementing like this would be to show most recently used.

**Reply by mk56-spn** [reply]
Sounds like an interesting option to have. As a sidenote do you have any opinion on the suggestion of shifting the locations of the aforementioned mods
